core/src/main/java/org/elasticsearch/action/admin/indices/rollover/RolloverResponse.java

```diff
@@ -95,6 +98,14 @@ public boolean isRolledOver() {
         return rolledOver;
     }
 
+    /**
+     * Returns true if the requisite number of shards were started before
+     * returning from the index creation operation.
+     */
+    public boolean isShardsAcked() {
```

Comment:
@bleskes the `RolloverResponse` seems to be missing fields because `rolledOver` only seems to indicate if `dryRun=false` and the conditions were met, but doesn't give any indication of the success of either the create index cluster state update operation nor the alias switch cluster state update operation.  I've added both in this commit: https://github.com/elastic/elasticsearch/pull/18985/commits/6c2e48a78369318330e5357d5c84d885c0a9423b